cypress/integration/infrastructure/not_served_by_Maestro_spec.js

```diff
@@ -0,0 +1,103 @@
+function responseShouldNotBeFromMaestro(response){
+  expect(response.headers['x-varnish']).not.to.exist;
+
+  if(response.headers.via != undefined){
+    expect(response.headers.via).not.to.contain('varnish');
+  }
+}
+
+function pageShouldNotBeFromMaestro(response){
+  expect(response.body).not.to.contain('meta name="Maestro"');
```

Comment:
We should probably test the positive versus the negative here

instead of seeing if it doesn't have Maestro for crds-net pages lets test to see if it has Netlify

This below code exists in _head.html in the crds-net project
`<meta name="Netlify" content="This page was rendered with Netlify">`

Reply:
Netlify check added.